graph-builder/src/status.rs

```diff
@@ -23,12 +23,12 @@ pub async fn serve_liveness(app_data: actix_web::web::Data<State>) -> Fallible<H
 ///
 /// Status:
 ///  * Ready (200 code): a JSON graph as the result of a successful scrape is available.
-///  * Not Ready (500 code): no JSON graph available yet.
+///  * Service Unavailable (503 code): no JSON graph available yet.
```

Comment:
The docstring is meant to describe the semantics of the result ("ready" or "not ready"), not the meaning of the HTTP code.

Reply:
To add to Luca's comment:

```suggestion
///  * Not Ready (503 code): no JSON graph available yet.
```